packages/spacecat-shared-data-access/src/index.d.ts

```diff
@@ -129,6 +129,47 @@ export interface AuditConfig {
   getAuditTypeConfigs: () => object;
 }
 
+/**
+ * Represents a key event.
+ */
+export interface KeyEvent {
+  /**
+   * Retrieves the ID of the key event.
+   * @returns {string} The Key Event ID.
+   */
+  getId: () => string;
+
+  /**
+   * Retrieves the site id of the key event.
+   * @returns {string} site id
+   */
+  getSiteId: () => string;
+
+  /**
+   * Retrieves the name of the key event.
+   * @returns {string} The name
+   */
+  getName: () => string;
+
+  /**
+   * Retrieves the type of the key event.
+   * @returns {string} The type
+   */
+  getType: () => string;
+
+  /**
+   * Retrieves the creation timestamp of the key event.
+   * @returns {string} The creation timestamp.
+   */
+  getCreatedAt: () => string;
+
+  /**
+   * Retrieves the last update timestamp of the key event.
+   * @returns {string} The last update timestamp.
+   */
+  getUpdatedAt: () => string;
```

Comment:
whats the purpose of having updatedAt for a Key Event if there is no way to update a key event? I would either add an update method to the API or remove this.

Reply:
`id`, `createdAt` and `updatedAt` come from the `Base` model. All data models have those fields by default since they all extend to the `Base` model: https://github.com/adobe/spacecat-shared/blob/main/packages/spacecat-shared-data-access/src/models/base.js#L35